go-controller/pkg/ovn/common.go

```diff
@@ -199,28 +198,6 @@ func (oc *Controller) deletePortGroup(hashName string) {
 	}
 }
 
-func (oc *Controller) getIPFromOvnAnnotation(ovnAnnotation string) string {
```

Comment:
you could keep this function and just make it be a wrapper around UnmarshalPodAnnotation... (the fewer places I see `.IP.IP.String()` the better...)

Reply:
@danwinship can we keep it for now? It goes away in the upcoming PR I split this cleanup off from...